hack/custom-boilerplate.go.txt

```diff
@@ -1,14 +1,3 @@
+// SPDX-License-Identifier: Apache-2.0
 // Copyright 2017-2021 Authors of Cilium
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
```

Comment:
Hm, is this the source for the generated files' license headers? If so, then shouldn't it be in the second commit instead of reverting changes from the first commit in the second for the generated files?

Reply:
This is not, my guess this is just a file ppl can cat into their source code to not have to remember the license header. 

The spdx conversation happens dynmically by the tool. @aanm pinged me to update this boiler plate file.

Reply:
ah, it appears you're right Paul:

```
define generate_k8s_api
	cd "./vendor/k8s.io/code-generator" && \
	GO111MODULE=off bash ./generate-groups.sh $(1) \
	    $(2) \
	    $(3) \
	    $(4) \
	    --go-header-file "$(PWD)/hack/custom-boilerplate.go.txt"
endef
```

I can update the boilerplate and then run the gen, then shove that into the "fixup" commit.